### ISSUE TITLE
Add .d.ts / linting guide

### ISSUE THREAD

**Issue by arlyon** [arlyon/stailwc#17]
The linter experience is not great atm as traditional tools don't really pick it up very easily. We would like to fix this.

**Comment by rutcreate** [comment]
@arlyon any plan on this?

**Comment by arlyon** [comment]
Yes, I have actually partially completed it (https://github.com/arlyon/stailwc/commit/b3cc83183a1085c783b79980a19592986e9da30b) just haven't released 0.14.1 yet :)

**Comment by rutcreate** [comment]
Cool! Can't wait

**Comment by arlyon** [comment]
New release out. I would like to expand on the 'optimal' vscode setup, but type checking is working. I will leave this issue open until I have more details on IDE integration.
